VogtPayroll6/Employee.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll6
+{
+    class Employee
+    {
+        private int empNumber;
+        private int empHoursWorked;
+        private decimal empPayrate;
+        private decimal empTaxRate;
+
+        public int EmpNumber { get => empNumber; }
+        public int EmpHoursWorked { get => empHoursWorked; }
+        public decimal EmpPayrate { get => empPayrate; }
+        public decimal EmpTaxRate { get => empTaxRate; }
```

Comment:
Why not get rid of the private variables and just have public variables with no setters? Less code to think about.